src/Microsoft.Android.Sdk.ILLink/MarkJavaObjects.cs

```diff
@@ -58,17 +58,18 @@ public void ProcessAssembly (AssemblyDefinition assembly, string androidHttpClie
 				if (!type.ImplementsIJavaObject (cache))
 					continue;
 
-				// Custom views in Android .xml files
+				// PreserveJavaObjectImplementation, but don't mark the type
+				PreserveJavaObjectImplementation (type);
```

Comment:
This might have gone too far, it made "hello world" grow a lot:
```diff
--  "PackageSize": 2865685
++  "PackageSize": 3213845
```
Going to try just the `IntPtr` constructor, which probably will end up where the original commit was.